src/3d/Viewport3d.tsx

```diff
@@ -157,6 +161,8 @@ const Viewport3d = () => {
                 </Plane>
             </Canvas>
             <Toolbar onZoomExtents={zoomExtents} onTogglePerspective={handleTogglePerspective} onNewFile={handleNewFile}></Toolbar>
+            <MobileButton onActivate={() => dispatch(toggleMobileMode())} />
+            {mobileModeActive && <MobileSelector cam={cameraTarget} onClose={() => dispatch(toggleMobileMode())} />}
```

Comment:
Mobile Button and Mobile Selector should probably exist in their own component now
```ts
<MobileTools/>
```